content/tutorials/pulumi-deployments-click-to-deploy/index.md

```diff
@@ -0,0 +1,120 @@
+---
+title_tag: Create and Manage cloud infrastructure with Pulumi Deployments | Learn Pulumi
+title: "Create Infrastructure with Pulumi Deployments Click-to-Deploy"
+layout: single
+description: |
+    Create infrastructure deployments using the New Project Wizard and GitHub integration to enable push-to-deploy in the Pulumi Cloud.
+meta_desc: In this step-by-step tutorial, learn how to create and manage cloud infrastructure using Pulumi Deployments, the New Project Wizard, and GitHub integration.
+meta_image: meta.png
+weight: 50
+summary: |
+    In this tutorial, you'll learn how to create, deploy, and manage cloud infrastructure using Pulumi Deployments with Click-to-Deploy. You'll start by using the New Project Wizard to set up and launch infrastructure from your browser, selecting a template to get started quickly. Additionally, you'll configure the Pulumi GitHub App to support additional deployment triggers, including push-to-deploy.
```

Comment:
How different is this than the current conceptual docs? Is this net new? Are we deprecating the old thing?

Reply:
Net-new. We don't have anything this deep; just a short paragraph here: https://www.pulumi.com/docs/pulumi-cloud/deployments/reference/#click-to-deploy. Would not be depreciating anything old.

Reply:
That's the plan, once complete I'll be redirecting so it's not two pages.